README.md

```diff
@@ -1,55 +1,70 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+**Description**: This repository contains the redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from WordPress and Django back-ends to Elasticsearch to render the site.
 
-### This project is a work in progress
+**Technology stack**:
```

Comment:
It's also interesting because this repo is mostly about Front-end stuff like Jinja templates, CSS and JS, but there is a huge dependency on things way more technical than that. There is some Python for the processors, but that's about it. Should this be noted as well?